### ISSUE TITLE
Not showing up

### ISSUE THREAD

**Issue by reptil1990** [garyjohnson/opentx-ble-joystick#1]
Hey awsome work!

I can get my QX7S connected but then I cant see it in freerider. I run openTX 2.2.1RC1!
Are there additional steps I have to make?

**Comment by garyjohnson** [comment]
Make sure you configure your model on the TX as described here: https://github.com/garyjohnson/opentx-ble-joystick#model-configuration

Then get Joystick Show and make sure a joystick appears and your TX is actually moving the joystick inputs: https://itunes.apple.com/us/app/joystick-show/id515886877?mt=12

Finally, if you use Liftoff, I'm having a hell of a time getting any joystick to work with it since they redid the input stack. FPV Freerider has been working great for me though.

**Comment by garyjohnson** [comment]
One more thing -- I'm not sure it's necessary yet, but if you just installed foohid, try giving your machine a reboot. I don't think it's needed but since it's a kernel extension I don't know if it loads right away.

**Comment by reptil1990** [comment]
Perfect will try that and come back to you!

**Comment by garyjohnson** [comment]
Also just noticed that Joystick Show is $1 -- any app you can find to test the joystick inputs independently of the sim would be useful for troubleshooting.

**Comment by reptil1990** [comment]
This 1$ is well spend! I appreciate your effort!

**Comment by reptil1990** [comment]
OK restart done. The Joystick Show is giving me the inputs corresponding to my RC inputs. But I can´t get it working in freerider or velocydrone

**Comment by garyjohnson** [comment]
When you move the sticks they move in Joystick Show? That should be all you need to get it working in Freerider. Are you launching Freerider on it's own or through something like Steam?

**Comment by reptil1990** [comment]
Yes they do... And i am launching FPV freerider as a standalone programm. not through steam or something else

**Comment by reptil1990** [comment]
So I tested Liftoff, FPV Freerider and Velocydrone all not working
The only one working was the Aerofly RC7 Sim!

**Comment by garyjohnson** [comment]
I'll have to try to reproduce. Any of this info you provide would be helpful:

macOS version
FPV Freerider version (Recharged?)
Are any other joysticks attached or show up in Joystick Show while you do this?

**Comment by garyjohnson** [comment]
Oh, and another thing -- do these games tell you there's no joystick attached, or does it appear but not accept input?

**Comment by reptil1990** [comment]
Osx 10.13.2 Beta
Latest version fpv freerider recharged
No other joysticks attached!
And the games say no input divination attached!

**Comment by garyjohnson** [comment]
Not quite sure what's going on yet. I'm going to play with making the HID descriptor match what OpenTX uses when you plug it in via USB and see if compatibility improves.

**Comment by reptil1990** [comment]
Let me know if you make an update I am happy to test

**Comment by garyjohnson** [comment]
Thanks! I've been trying to find the common denominator -- the app appears to work fine against both GameController.framework and the Unity Input API. Going to try SDL2 and some other possible gamepad APIs and see if I can figure out what these games might be using.

**Comment by reptil1990** [comment]
Anything new? Would be a waste if the project goes down the drain!
